src/util/session.ts

```diff
@@ -7,7 +7,7 @@ import * as vscode from "vscode";
 import { ChildProcess, spawn } from "child_process";
 
 function getGitHubURLFromOutput(output: string): string | null {
-  const remoteRegex = /origin\s+git@github.com:(.+?)\.git\s+\(fetch\)/;
+  const remoteRegex = /origin\s+https:\/\/github.com\/(.+?)\.git\s+\(fetch\) | origin\s+git@github.com:(.+?)\.git\s+\(fetch\)/;
```

Comment:
```suggestion
  const remoteRegex = /origin\s+(git@github.com:|https:\/\/github.com\/)(.+?)\.git\s+\(fetch\)/;
```

Actually for proper conditional regex, you need some brackets :)